### DISCUSSION TITLE
UX: Date selection shows year range from 1970, if something else is selected first

### DISCUSSION THREAD

**Discussion by azharul-islam** [directus/directus#5641] [Feature Requests]
![2021-05-12 21 12 29](https://user-images.githubusercontent.com/38819502/118024069-eed67400-b366-11eb-9098-28c1aee3201f.gif)

Version: rc62

If however, year is selected first, we get to select 2021 first. Also, I feel overall, the UX of date selection could use some improvement.

**Comment by rijkvanzanten** [comment]
Fixed in https://github.com/directus/directus/pull/5534

> Also, I feel overall, the UX of date selection could use some improvement.

Feel free to share any ideas / optimizations / code you have, this is OSS after all. Just saying "could use improvement" doesn't really help anybody 🙂

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
